source/_posts/2020-03-14-2020-using-vyper-with-embark.md

```diff
@@ -0,0 +1,204 @@
+title: Using Vyper with Embark
+summary: "Vyper has gained some traction on Solidity as an Ethereum smart contract language, especially with support going in to ETH2. Learn how to use Embark to use Vyper smart contracts in a &ETH;App."
+author: eric_mastro
+categories:
+  - Report
+layout: blog-post
+image: '/assets/images/vyper_simple_storage_with_logo.png'
+---
+
+![Vyper Language](/assets/images/vyper_simple_storage_with_logo.png)
+
+## Why Vyper?
+[Vyper](https://github.com/vyperlang/vyper) is a contract-based approach to writing Ethereum smart contracts, and is aimed at bringing ["increased security and reability through a minimalist approach"](https://ethereumdevelop.com/ethereum-solidity-vyper/) to the smart contract landscape.
+
+Although Vyper has gained some popularity in recent times as a security-focused alternative to Solidity, the simplicity required for such a case [leaves out some features of Solidity](https://ethereumdevelop.com/ethereum-solidity-vyper/) that developers have come to appreciate.
+
+In its current form, Vyper is python-based, but due to the [Consensys audit highlighting some major security flaws](https://diligence.consensys.net/audits/2019/10/vyper/), the Ethereum Foundation has decided to concurrently build a [Rust-based Vyper](https://github.com/ethereum/rust-vyper), while community devs aim to support the original Python-based version.
+
+{% notification danger 'Using Vyper in production' %}
+Because of the inherent security flaws, please **do not** use Python-based Vyper in production projects until the [vyperlang team](https://github.com/vyperlang/vyper) has created a production-ready release. Embark will also support Rust-based Vyper once it is released.
+{% endnotification %}
+
+Irrespective of the security audit, [Vyper is perhaps the most intuitive and easiest to quickly learn language for reading and writing Etheruem contracts - even without any prior knowledge or experience in programming](https://www.publish0x.com/rhyzom/vyper-compiler-security-audit-and-developing-a-new-rust-base-xxpygm). Additionally, Embark has created a plugin that makes it just as easy to compile Vyper contracts in an Embark &ETH;App!
```

Comment:
I think we can just make the `[...] the most intuitive and easiest to quickly learn language [...]` part the link.

Reply:
Was just trying to avoid plagarising, but ok.